examples/simple-email-proof/vlayer/src/shared/hooks/useEmailProofVerification.ts

```diff
@@ -106,15 +107,20 @@ export const useEmailProofVerification = () => {
     writeContract(contractArgs);
   };
 
+  const [preverifyError, setPreverifyError] = useState<Error | null>(null);
   const startProving = async (emlContent: string) => {
     setCurrentStep(ProofVerificationStep.SENDING_TO_PROVER);
 
-    const email = await preverifyEmail({
-      mimeEmail: emlContent,
-      dnsResolverUrl: import.meta.env.VITE_DNS_SERVICE_URL,
-      token: import.meta.env.VITE_VLAYER_API_TOKEN,
-    });
-    await callProver([email]);
+    try {
+      const email = await preverifyEmail({
+        mimeEmail: emlContent,
+        dnsResolverUrl: import.meta.env.VITE_DNS_SERVICE_URL,
+        token: import.meta.env.VITE_VLAYER_API_TOKEN,
+      });
+      await callProver([email]);
+    } catch (error) {
+      setPreverifyError(error as Error);
+    }
```

Comment:
Do we have a plan for testing fallible paths in playwright? I thought we did something with that in mind in `test-web-app` but I might be misremembering. Anyhow, would it be useful to include in testing examples also? cc @wgromniak2

Reply:
BTW @aajj999 we should verify that we handle this error equally well in other examples.